seed/src/lib.rs

```diff
@@ -152,11 +162,11 @@ impl Node {
             protocol: protocol::Config {
                 paths,
                 listen_addr: config.listen_addr,
-                membership: Default::default(),
+                membership: config.membership.clone(),
```

Comment:
It's a bit odd to first consume the values, but then keep them around until `run`, which consumes `self` and obviously doesn't need those protocol parameters anymore.

Perhaps `peer_config` should just be built in `run`? Or `NodeConfig` should be consumed and `Node` should keep some other type with only the things it still needs?

Reply:
Ya, it feels weird indeed. Sorry I was cutting corners.

I think I'd like to make life easier for us here by moving the `radicle-bins/seed` over here and just have the exe there -- like you suggested. This dance is tiring. It would help with recognising these things, I think.